### ISSUE TITLE
navbar-utility wraps to two lines when status icon is present at viewport sizes 768px - 816px

### ISSUE THREAD

**Issue by rhamilto** [openshift/origin-web-console#197]
![screen shot 2016-07-20 at 4 42 42 pm](https://cloud.githubusercontent.com/assets/895728/17002440/480b866a-4e99-11e6-9dfb-89ef669584b3.PNG)

I noticed this while working on https://github.com/openshift/origin-web-console/pull/196.  Origin doesn't include a status icon, but you can mock one in browser with the following markup:

``` 
<li extension-point="" extension-name="nav-system-status" extension-types="dom" class="ng-isolate-scope dropdown"><a href="#" class="nav-item-iconic dropdown-toggle"><span class="fa pficon-warning-triangle-o"></span></a></li>
```

**Comment by spadgett** [comment]
Fixed by #196

**Comment by rhamilto** [comment]
This one is related to #196, but is not fixed by it.  Sorry for the confusion.
